src/core/test/system/base_system.sol

```diff
@@ -53,21 +53,31 @@ contract BaseSystemTest is TestSetup, Math, DSTest {
         rootAdmin.relyLenderAdmin(address(this), senior_);
     }
 
-    function createTestUsers() public {
-            borrower = new Borrower(address(shelf), address(lenderDeployer.distributor()), currency_, address(pile));
+    function createTestUsers(bool senior_) public {
```

Comment:
It would have been easier by adding just adding.
```javascript
function createTestUsers() public {
  bool deploySenior = false;
  createTestUsers(deploySenior);
}
```

both ways are fine I think

Reply:
yes, agree with Manuel. 
Also helos to avid merge conflicts